_sass/blocks/state-pages/_iconic-nav.scss

```diff
@@ -0,0 +1,76 @@
+.state-page-nav-title {
+  @include heading(large-caps);
+
+  align-items: center;
+  border-bottom: 1px solid $mid-gray;
+  display: flex;
+  justify-content: flex-start;
+  margin: 0;
+  position: relative;
+
+  &:hover,
+  &:focus
+  &:active {
+    svg.map .states.features {
+      fill: $blackest-black;
+      fill-opacity: 1;
+      stroke: $blackest-black;
+    }
+  }
+
+  svg.map {
+    .state.feature {
+      fill: $blackest-black;
+      fill-opacity: 1;
+    }
+
+    .states.features {
+      fill: $neutral-gray;
+      fill-opacity: 1;
+      stroke: $neutral-gray;
+      @include transition(all, 0.3s);
+    }
+  }
+
+  figure {
+    align-self: flex-end;
+    margin: 0;
+    width: 33%;
+
+    &:hover,
+    &:focus
+    &:active {
+
+      &+ .nav-prompt {
+        visibility: visible;
+      }
+
+      svg.map .states.features {
```

Comment:
The specificity on the style I was overriding was quite high, so it was either this or `!important`